### DISCUSSION TITLE
New GitHub GraphQL API Explorer is terrible

### DISCUSSION THREAD

**Discussion by Timmmy-nlb** [community/community#72715] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

The new GitHub GraphQL API Explorer is terrible,
because before you could use the explorer-tab to build the queries via GUI and fill the input fields without knowing the structure.
Now there is only the API docu without any interactive part that builds the query itself. 
This is an extreme loss of comfort and time saving!

PS: To get the explorer without the frame as a separate way window would be great.

![grafik](https://github.com/community/community/assets/52195132/454b6c21-b574-4e3f-a5c7-702b89f06fff)

**Comment by ghostinhershell** [comment]
Hi @Timmmy-nlb, 

Thanks so much for sharing your feedback with the community. We actually made the decision to sunset the feature given it doesn't meet our accessibility standards. We opened an [issue](https://github.com/graphql/graphiql/issues/3422) with the maintainers of the repo to potentially have those accessibility concerns addressed, but in the meantime, we will not be supporting the plugin.

**Reply by cringdahl** [reply]
I'm seeing there's been a reply to the issue, specifically asking for a repeat audit to their v2. I, too, rely on the explorer as I am utterly unfamiliar with GraphQL in general, but respect the power of it over the REST API. As an Enterprise customer, I've worked with a member of your staff who also uses the feature to build out his queries.

Any priority that can be given to this would be greatly appreciated; I need to be able to provide useful Github metrics to my teams, and the GraphQL explorer cuts my dev time from days to hours.

**Reply by nul800sebastiaan** [reply]
Oh the irony of having a accessibility tooling telling you that you're covered while the interface is now much harder to use for sighted people.. 

Please consider at the VERY least to make this available as a full screen page instead of putting it into this tiny, unusable iframe. 

And then consider the loss of productivity after disabling that plugin. I am a huge supporter of accessibility and if you really wanted to help then you'd send the plugin developer a pull request instead of just yanking this most useful tool, or make a plugin of your own. 

Is there an option to opt-in to the old editor so people at least get to choose?

**Reply by nul800sebastiaan** [reply]
FYI: while not as good (and definitely with some performance issues) you can set something similar up in Postman, and probably other API-related apps as well. 

![image](https://github.com/community/community/assets/304656/a86b38f3-6bdd-40d7-8587-9750c1db2edb)

**Comment by Timmmy-nlb** [comment]
It is regrettable not to hear that there are such internal fronts and disputes that prevent you from delivering a good product.

**Comment by noqcks** [comment]
Someone created a CSS patch to enlarge the (terribly tiny) explorer window

https://www.cazzulino.com/gh-graphql.html

**Comment by Timmmy-nlb** [comment]
"Very great", you could have done that as early as October 2023:
https://github.blog/changelog/2025-08-21-graphql-explorer-removal-from-api-documentation-on-november-1-2025/
